app/javascript/packs/application.js

```diff
@@ -3,4 +3,6 @@ require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
 require("bootstrap/dist/js/bootstrap")
-require("../stylesheets/application.scss")
\ No newline at end of file
+require("../stylesheets/application.scss")
+require('jquery')
+require("movie_search.js")
```

Comment:
pack内のファイルは `./` が必要です。

```suggestion
require("./movie_search.js")
```